utils/file.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+)
+
+// ChmodExecutable sets the executable mode/flag on a file, if not already.
+func ChmodExecutable(filename string) error {
```

Comment:
Feels on renaming this to `SetChmodExecutable`? Reading the code that uses it, I can either interpret it as a "does this file have chmod executable" or "set this as chmod executable"

Reply:
My thinking was “chmod” is short for “change mode”, so it implies the verb.

Reply:
Oh yeah, doh! Good point.